src/package-managers/staticRegistry.ts

```diff
@@ -0,0 +1,35 @@
+import fs from 'fs'
+import { GetVersion } from '../types/GetVersion'
+import { Version } from '../types/Version'
+import { Options } from '../types/Options'
+import { StaticRegistry } from '../types/StaticRegistry'
+
+let registry: StaticRegistry = {}
+
+/**
+ * Assigns static registry dependencies to variable given a valid path
+ *
+ * @param path
+ * @returns
+ */
+const initializeRegistry = (path: string | undefined): void => {
+  if (path === undefined || !fs.existsSync(path)) {
+    throw new Error('No or invalid path to static registry was provided. Please try again')
+  }
```

Comment:
I'd like to separate these two checks in order to provide more useful feedback to the user.

Please use [programError](https://github.com/raineorshine/npm-check-updates/blob/d7df1515391ce2084fd73bd4d8b55b09fa57ae1c/src/lib/programError.ts) for errors with known causes rather rather than throwing an uncaught exception. This will make it clear that it is a user error, not a bug in npm-check-updates.

1. The missing `options.registry` check should go in [initOptions](https://github.com/raineorshine/npm-check-updates/blob/main/src/lib/initOptions.ts) where other options validation lives. Rather than `Please try again`, say `When --package-manager staticRegistry is specified, you must provide the path for the registry file with --registry. Run "ncu --help --packageManager" for details.`

2. Change "invalid path" to say "The specified static registry file does not exist: [FILE]".